tests/bson/bson-utcdatetime-001.phpt

```diff
@@ -11,7 +11,7 @@ require_once __DIR__ . "/../utils/basic.inc";
 
 $manager = create_test_manager();
 
-$utcdatetime = new MongoDB\BSON\UTCDateTime("1416445411987");
+$utcdatetime = new MongoDB\BSON\UTCDateTime(2147483647);
```

Comment:
Why did you opt to use `INT32_MAX` here instead of construct an Int64? If you wrap this value in an Int64, that should allow the expected output below to remain as-is.

Reply:
I initially changed all tests to `INT32_MAX` before realising that using an `Int64` instance was better, but forgot to change these two tests. These are now consistent with the others.